docs/model.md

```diff
@@ -190,3 +190,14 @@ The model may have a model repository (initiated by some scope provider or by
 the metamodel). This object is responsible to provide and cache other model
 instances (see textx.scoping.providers).
 
+### _tx_model_params
+
+This attribute always exists. It holds all additional parameters passed to
+`model_from_str` or `model_from_file` of a metamodel. These parameters are
+restricted by the `metamodel._tx_model_param_definitions` object, which is
+controlled by the metamodel designer. 
+
+`metamodel._tx_model_param_definitions` can be queried (like a dict) to
+retrieve possible extra parameters and their description for a metamodel.
+It is also used to restrict the additional parameters passed to
+`model_from_str` or `model_from_file`.
```

Comment:
We should extend this section (or maybe add something in the metamodel docs) to specify how new params are defined and in obj. processors for example.

Reply:
Done.

Reply:
Added an example and a unittest.